site-src/contributing/release-cycle.md

```diff
@@ -0,0 +1,104 @@
+# Release Cycle
+
+In Gateway API 1.2+, we will be following a more structured and predictable
+release cycle that is inspired by the [upstream Kubernetes release
+cycle](https://kubernetes.io/releases/release/).
+
+## Goals
+
+* Ensure a predictable release schedule that enables 2-3 releases a year
```

Comment:
imo 2-3 itself is vague
can we add more clarity here ? e.g.
* https://github.com/envoyproxy/envoy/blob/main/RELEASES.md#major-release-schedule
* https://istio.io/latest/docs/releases/supported-releases/#support-policy
* https://gateway.envoyproxy.io/announcements/#release-schedule

Reply:
That's definitely my longer term goal, but I think we need to try at least one of these structured releases before we can confidently predict dates with that level of precision. The current goal is to set dates for a minimum of 1 phase in advance, but ideally more. Hopefully once we have some experience, we'll be able to set dates for every phase in a release cycle upfront, matching upstream Kubernetes. The biggest thing that could help us would be having people that can help push things forward, especially with this release process.